### PR TITLE
📝 Adds base mac setup md file

### PR DESCRIPTION
**Pull request by BrentWMiller**
## Task(s)
Add mac setup guide

## What was updated?
Pulls in content from https://www.craft.do/s/zJLV3mHIrD9euT

- [ ] Properly formatted and cleaned up
- [ ] Implement Ryan's comments

## Screenshots of updates (visual changes)
**none**

### CODE REVIEW

File: mac-setup.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- [ ] Install Drush globally via Composer [https://docs.drush.org/en/8.x/install-alternative/](https://docs.drush.org/en/8.x/install-alternative/)
+
+   Add the following to `.zshrc`
+
+```Bash
+alias mysql=/Applications/MAMP/Library/bin/mysql
+export PATH="/Applications/MAMP/Library/bin:$PATH"
+```
+
+- [ ] Set up PHP path to go to MAMP [https://stackoverflow.com/questions/4145667/how-to-override-the-path-of-php-to-use-the-mamp-path/10653443#10653443](https://stackoverflow.com/questions/4145667/how-to-override-the-path-of-php-to-use-the-mamp-path/10653443#10653443)
```
Comment (BrentWMiller) [inline mac-setup.md:69]: > I like the answer below it even better https://stackoverflow.com/a/29990624

- Ryan's comment

File: mac-setup.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### Web
+
+- [ ] Install MAMP Pro
+   - [ ] Keys in 1Password or recover via email
+   - [ ] Set server ports to 80, 81, etc (use button) in "Ports & Users"
+   - [ ] Turn off cache module in "PHP"
+   - [ ] Install Sequel Ace in "MySQL"
+   - [ ] Enable network access to SQL in "MySQL" in order to use Sequel Ace
+- [ ] Install Composer [https://getcomposer.org/download/](https://getcomposer.org/download/) make sure you global install with the `mv` command on that page
```
Comment (BrentWMiller) [inline mac-setup.md:56]: > In order to have php in the command line you need to enable it on the PHP tab in MAMP.  https://cln.sh/gcFPHF

- Ryan's comment

**Review by BrentWMiller — COMMENTED** [review]
Added Ryan's comments from the Craft doc

**Comment by ryanschmidt** [comment]
I like this. Can you add the link to the README?

**Comment by ryanschmidt** [comment]
@BrentWMiller

**Comment by BrentWMiller** [comment]
Totally forgot about this. I'll hit this today

**Pull request merged**